conf/broker.conf

```diff
@@ -1769,6 +1766,12 @@ managedLedgerOffloadPrefetchRounds=1
 # Use Open Range-Set to cache unacked messages
 managedLedgerUnackedRangesOpenCacheSetEnabled=true
 
+# Whether persist cursor ack stats as long arrays, which will compress the data and reduce GC rate
+# When it's false, the behavior will be the same with 3.x or earlier
+# Modifying this config could lose existing individual acknowledgments, so you should configure it with false when
+# upgrading from 3.x to 4.0 or later if you don't want to lose these acknowledgments.
+managedLedgerPersistIndividualAckAsLongArray=true
```

Comment:
There is a conclusion that we will never set the default value of `managedLedgerPersistIndividualAckAsLongArray ` to `true` for `branch-4.0.x`
- see details here : https://github.com/apache/pulsar/pull/23888#issue-2808801657

Reply:
Updated